flowmachine/flowmachine/features/subscriber/unique_values_from_queries.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Given a list of queries and a list of columns, returns a table of each unique combination of `column` .
+"""
+
+from typing import List
+
+from flowmachine.core.query import Query
+
+
+class UniqueValuesFromQueries(Query):
+    """
+    Class representing unique values in  across a set of queries
+
+    Parameters
+    ----------
+    query_list: List[Query]
+        A list of Query objects, each of which must include a `subscriber` column
+
+    """
+
+    def __init__(self, query_list: List[Query], column_names: List[str]):
+        self.query_list = query_list
+        self._column_names: List[str] = column_names
```

Comment:
Probably a good idea to check here that all of the queries in `query_list` include the requested columns (based on their `column_names` properties).

Reply:
There a specific exception in FM for this situation, or just raise `ValueError`?

Reply:
Ah, just found `MissingColumnsError` seems like it could be a good fit

Reply:
Ah, good find! I couldn't find an example that raises anything other than `ValueError`, but `MissingColumnsError` looks good (although it looks like it's not an exception we're currently using anywhere in the codebase...)

Reply:
(Apparently I can't start a second comment thread against the same line of code, so I'll unresolve this one to add a new comment)

Might be good to do `self._column_names = list(column_names)` here, to enforce the same internal attribute regardless of the type of iterable passed in.

Reply:
Oh, wait, I can do it now! I'll move this comment to the appropriate place.